testCafe/questions/radiogroup.js

```diff
@@ -74,13 +74,13 @@ frameworks.forEach(framework => {
 
     //asc
     await setOptions("car", { choicesOrder: "asc" });
-    await t.expect(chocies.nth(0).find("[aria-label=\"Audi\"]").exists).ok();
-    await t.expect(chocies.nth(3).find("[aria-label=\"BMW\"]").exists).ok();
+    await t.expect(chocies.nth(0).find("[type=\"radio\"]").exists).ok();
```

Comment:
It looks like here we checked items order using aria label. Changes Audi/BMW to radio/radio break the test. Probably other checks should be also validated.